contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -363,8 +363,8 @@ PRAGMA_WARNING_DISABLE_VS(4355)
 				}
 				
 				delay *= 0.5;
-				if (delay > 0) {
-					long int ms = (long int)(delay * 100);
+				long int ms = (long int)(delay * 100);
```

Comment:
How was this going negative previously? Overflow?

Reply:
I don't know for sure. binaryFate reported a negative delay, but I'm not sure exactly when it was from, so I armoured the places that call the delay. I made a program to check the ranges with those types and this seems it should be OK, if the throttle code returns a sane number, but I decided to armour anyway.